src/main/webapp/data/userMapData.json

```diff
@@ -1575,6 +1623,7 @@
       "Marshall B. Ketchum University",
       "Mary Ellen Henderson",
       "Marywood University",
+      "Maryville University St. Louis",
```

Comment:
can delete this. it is the same as `Maryville University`

Reply:
I think you missed out on this! Both names direct to the same university so I think can remove.

Reply:
Where is the other Maryville? There's Marywood but that's not the same University

Reply:
ohh i see. My bad on this!